server/src/main/scala/actors/queues/DateRange.scala

```diff
@@ -0,0 +1,29 @@
+package actors.queues
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import drt.shared.{MilliTimes, SDateLike, UtcDate}
+import services.SDate
+
+object DateRange {
+
+  def utcDateRangeWithBuffer(start: SDateLike, end: SDateLike): Seq[UtcDate] = {
```

Comment:
Wondering if this should be a curried function to pass in the -2 & +1 rather than have them hard coded?